src/seedu/addressbook/AddressBook.java

```diff
@@ -203,13 +203,17 @@ public static void main(String[] args) {
         processProgramArgs(args);
         loadDataFromStorage();
         while (true) {
-            String userCommand = getUserInput();
-            echoUserCommand(userCommand);
-            String feedback = executeCommand(userCommand);
-            showResultToUser(feedback);
+            getAndExecuteCommand();
```

Comment:
as discussed in the tutorial session, it would make more sense to include the loop as well in the extracted method